### PR TITLE
fix: Remove ctrl/cmd+click text and functionality 

### PR DESCRIPTION
**Pull request by mathur07**

### CODE REVIEW

**Review by sawood14012 — APPROVED** [review]
LGTM

File: src/app/stack/card-details/report-information/report-information.component.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
             <b>Register</b> for a <b>free</b> Snyk account to find out more, including whether any of them have a publicly known exploit.
         </span>
         <span class="col-sm-2">
-            <a [href]="this.generateUrl.registrationURL" target="_blank" title="command/ctrl + click"
+            <a [href]="this.generateUrl.registrationURL" target="_blank"
                 (click)="this.handleUrl.open($event, this.generateUrl.registrationURL)"><button class="ripple">Sign up for
```
Comment (arajkumar) [inline src/app/stack/card-details/report-information/report-information.component.html:15]: Shouldn't we do preventDefault on the click event?

Comment (mathur07) [reply src/app/stack/card-details/report-information/report-information.component.html:15]: Ya correct, I forgot to do that. Added it now.

**Review by arajkumar — CHANGES_REQUESTED** [review]
There is a garbage file " Paste doesn't activate token Submit button". Please remove.
https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/blob/c2bc95252f59e352bd4f44705e8537ee49d6136c/Paste%20doesn't%20activate%20token%20Submit%20button

**Comment by mathur07** [comment]
@arajkumar yes, I have removed the garbage file.

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**